caustic/lenses/base.py

```diff
@@ -7,7 +7,7 @@
 from torch import Tensor
 
 from ..constants import arcsec_to_rad, c_Mpc_s
-from ..cosmology import Cosmology
+from ..cosmology.base import Cosmology
```

Comment:
Instead of adding `.base` to all of the imports, we can probably use a `__init__.py` file to make the modules importable from `..cosmology`

Reply:
Fixed! I just forgot to make that change after adding the __init__.py earlier.